packages/solo/src/start.js

```diff
@@ -444,8 +444,12 @@ export default async function start(basedir, argv) {
       if (err) {
         console.error(err);
       }
+      // eslint-disable-next-line no-use-before-define
+      process.off('exit', killDeployment);
     },
   );
+  const killDeployment = () => cp.kill('SIGINT');
+  process.on('exit', killDeployment);
```

Comment:
This seems unrelated to this issue.

Reply:
It helped me debug problems while creating this PR.  Would you prefer I remove it?

Reply:
I think there is a more general issue with children management (e.g. the cli tends to leave its child behind too). I personally prefer keeping PRs focused, but I'll defer on best practices for now. If included, it should probably be its own commit tho.